nujo/utils/data/dataset.py

```diff
@@ -0,0 +1,42 @@
+from os.path import expanduser
+
+from numpy import array, empty, float32, vstack
+
+from nujo.utils.data.dataset_iterator import DatasetIterator
+
+
+class Dataset:
+    '''
+
+    Parameters:
+    -----------
+    name : will be downloaded from the UCI ML repo
+
+    Returns:
+    --------
+    tuple : stores the csv dataset,
+    - np array with floating point integers
+    - np array with labels
+    '''
+    def __init__(self, name):
+        file = expanduser('~/.nujo/') + name
```

Comment:
That's why you should export it in a separate file and import it from the file when needed

Reply:
what do you mean

Reply:
`expanduser('~/.nujo/')` is used both here and data_loader.py, so its better if you make a separate file where you assign `expanduser('~/.nujo/')` to a const and import it from there when needed